home-tasks/src/test/java/com/epam/tc/hw2/utils/LogIn.java

```diff
@@ -0,0 +1,39 @@
+package com.epam.tc.hw2.utils;
```

Comment:
Please move to steps package
```suggestion
package com.epam.tc.hw2.steps;
```